src/helpers.py

```diff
@@ -59,26 +59,22 @@ def get_collateral_token_range(
     collateral_token_underlying_address: str,
     collateral_token_price: float,
 ) -> list[float]:
-    # TODO: improve
-    STEPS = {
-        "0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7": 50.0,  # ETH
-        "0x03fe2b97c1fd336e750087d68b9b867997fd64a2661ff3ca5a7c771641e8e7ac": 1_000.0,  # WBTC
-        "0x04718f5a0fc34cc1af16a1cdee98ffb20c31f5cd61d6ab07201858f4287c938d": 0.01,  # STRK
-    }
     target_values = 50 
-    start_price = STEPS[collateral_token_underlying_address]
+    start_price = collateral_token_price
```

Comment:
We want the ~50 steps to cover the range between 0.0 and `1.2 * collateral_token_price`, not between `collateral_token_price` and `1.2 * collateral_token_price`. But 0 shouldn't be included, in order not to mess up the computations (so set `start_price` at 0.0 and then drop the initial 0.0 from the final range).